Tests/DependencyInjection/GoAopExtensionTest.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2015, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Symfony\GoAopBundle\Tests\DependencyInjection;
+
+use Go\Symfony\GoAopBundle\DependencyInjection\GoAopExtension;
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractExtensionTestCase;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * Class GoAopExtensionTest
+ */
+class GoAopExtensionTest extends AbstractExtensionTestCase
+{
+    /**
+     * @test
+     */
+    public function itLoadsServices()
+    {
+        $this->load();
+
+        $services = [
+            'goaop.aspect.kernel',
+            'goaop.aspect.container',
+            'goaop.cache.path.manager',
+            'goaop.cache.warmer',
+            'goaop.command.warmup',
+            'goaop.command.debug_advisor',
+            'goaop.command.debug_aspect',
+            'goaop.bridge.doctrine.metadata_load_interceptor'
+        ];
+
+        foreach ($services as $id) {
+            $this->assertContainerBuilderHasService($id);
+        }
+
+        $this->assertEquals(8, count(array_filter($this->container->getDefinitions(), function ($id) {
```

Comment:
It will be better to replace the magic number "8" with `count($services)` and I can also suggest to rename `$services` variable to the `$expectedServices` for better code reading.